mm_image_processing/server.py

```diff
@@ -9,16 +9,18 @@
 import json
 import asyncio
 import websockets
-import util.image_processor as image_processor
 
 from threading import Thread, Lock
-from util.s3_img_getter import getImg
 from flask_sock import Sock
 from flask import Flask, request, make_response
 from flask_cors import CORS
-from util.image_processor import processImgFromLocal
-from util.s3_img_getter import getImg, deleteImg
 
+# Ultility functions, using relative imports here since we are using a poetry entrypoint to run our Flask server
+# Using absolute imports results in Module not found errors whenever we try to call functions from the util directory
+from .util import image_processor
+from .util.s3_img_getter import getImg
+from .util.image_processor import processImgFromLocal
+from .util.s3_img_getter import getImg, deleteImg
```

Comment:
The way this repo is structured, your package provides the following modules: `mm_image_processing` and `util`.

I would recommend that the `util` module be a submodule of `mm_image_processing`.  In order for any of these to be proper python modules, `util` must also have a `__init__.py`, and be referenced as `mm_image_processing.util`

Reply:
Updated to follow the above format.